Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -90,27 +90,23 @@ def elabMeasure (μ : TSyntax `term) (expectedType? : Option Expr) : TermElabM E
     let _ ← mkFreshExprMVarQ q(MeasurableSpace $ty)
     elabTerm μ q(Measure $ty)
 
+/-- Try to elaborate `μ` as a term of type `T` where `OuterMeasureClass T ?Ω`. If that fails, try to
+elaborate `μ` as `Measure ?Ω`. -/
+elab "elab_as_measure% " t:term : term => do elabMeasure t ‹_›
```

Comment:
I'm surprised `‹_›` works here!

It might be better to instead put `@[term_elab elab_as_measure]` on the `def` above and use
```suggestion
syntax (name := elab_as_measure) "elab_as_measure% " t:term : term
```
which also shortcuts some expected type logic you might not need.

Reply:
I looked at the context and saw there was an inaccessible `Option Expr` 🕵️

Reply:
```
@[term_elab elab_as_measure]
def elabMeasure (μ : Syntax) (expectedType? : Option Expr) : TermElabM Expr := do
```
errors with
```
unexpected type at 'Mathlib.Meta.elabMeasure', `Lean.Elab.Term.TermElab` expected
```